sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2072,6 +2072,10 @@ def check_resources_fit_cluster(self, handle: ResourceHandle,
 
         The resources requested by the task should be smaller than the existing
         cluster.
+
+        Raises:
+            exceptions.ResourcesMismatchError: If the resources requested by the
+                task are larger than the existing cluster.
```

Comment:
when we raise this error, is it necessary to be "larger"? or mismatch such as `accelerator: V100` vs `accelerator:A100`?

Reply:
Oops, good catch! Fixed. Thanks!